src/screens/dashboard/MyProfile.jsx

```diff
@@ -62,19 +78,31 @@ function MyProfile () {
             showPhoneVerificationModal &&
               <VerificationModal type='phone' clickHandler={phoneClickHandler} />
             }
-            <DashboardLayout title='My Profile'>
+            <DashboardLayout
+              title={
+              pathname === '/myProfile'
+                ? 'My Profile'
+                : 'Community Members'
+            }
+            >
               <div className='x10-4-0-my-personals'>
                 <div className='flex-col-2'>
                   <div className='frame-2923'>
-                    <BackButton location='/dashboard' />
+                    <BackButton onClick={() => history.goBack()} />
                   </div>
                   <div className='profile border-1px-onyx'>
                     <div className='profile-info'>
-                      <PersonalInformation user={user} />
-                      <ContactInformation user={user} verification={verification} />
-                      <AdditionalInformation user={user} />
+                      {user && <>
+                        <PersonalInformation user={user} />
+                        {user?.phone === undefined || <ContactInformation user={user} />}
+                        <AdditionalInformation user={user} />
+                      </>}
                     </div>
-                    <EditInformation clickHandler={editUserInformation} image={user?.attachments && process.env.REACT_APP_CDN_BASE_URL + '/attachments/' + user.attachments} />
+                    <EditInformation
+                      clickHandler={editUserInformation}
+                      image={user?.attachments && process.env.REACT_APP_CDN_BASE_URL + '/attachments/' + user.attachments}
+                      follow={pathname !== '/myProfile'}
```

Comment:
follow should not show if logged in user and current profile is not same irrespective of url